canonical-web-and-design/vanilla-framework
# Remove details doc page

## Done

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/2584

## QA

- Pull code
- Run `./run serve --watch`
- Verify details page is no longer in the nav
